README.md

```diff
@@ -35,6 +35,10 @@ or `skopeo`:
   * `quay.io/containers/*:latest` and `quay.io/*/stable:latest` -
     Built daily using the same `Containerfile` as above.  The tool versions
     will remain the "latest" available in Fedora.
+  * `quay.io/containers/aio:latest` and `quay.io/containers/aio:<date stamp>` -
+    Built weekly using a similar `Containerfile` as the Podman and Buildah
+    images.  It's a smaller, minimal image intended to be used as a base-image
+    for development containers or CI automation.
```

Comment:
I hit 'aio' here first and was a bit lost until I saw the README below.  I think a slight explanation will be helpful here.
```suggestion
    for development containers or CI automation.  It contains the Buildah, Podman, and Skopeo images "All In One".
```